homeassistant/components/onewire/sensor.py

```diff
@@ -234,20 +219,6 @@ def get_sensor_types(device_sub_type):
     return DEVICE_SENSORS
 
 
-async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
```

Comment:
Our standard is to log a warning for two releases after the deprecation PR and then remote support completely in a second invalidation PR. Have we logged a warning yet? Did we announce the deprecation already in a release note for a previous release?

Reply:
Config flow was added back in 0.118 (#39321) and removed from the documentation in https://github.com/home-assistant/home-assistant.io/pull/14238. I assumed it was enough time but don't think a specific warning was added or an official announcement made.

Is there a specific recommended wording for the deprecation PR?

Reply:
For the breaking change section see eg:
https://github.com/home-assistant/core/pull/50078

For the log warning in `async_setup_platform` see eg:
https://github.com/home-assistant/core/pull/46964

Let's make the deprecation explicit in this PR and then remove support completely after two releases.

Reply:
PR updated...